integration/indexes_command_test.go

```diff
@@ -598,3 +599,323 @@ func TestDropIndexesCommandInvalidCollection(t *testing.T) {
 		})
 	}
 }
+
+func TestListIndexesCommand(t *testing.T) {
```

Comment:
Added this test because MongoDB returned index fields in different order for `listIndexes` and `reIndexes` command which makes comparison difficult. https://github.com/FerretDB/FerretDB/pull/4735/files#diff-5ade7032f8efaf774bbedcb260b7bdde37af28384d81c26bfc7b52b78f909f36R785
>// For MongoDB, `unique` is the 2nd field of `reIndex` command, but it's the 4th field of `listIndexes` command.
						// For FerretDB, `unique` is the 4th field in both `reIndex` and `listIndexes` commands.